src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -651,9 +651,7 @@ TEST_CASE("basic contract invocation", "[tx][soroban]")
     }
 }
 
-#ifdef ENABLE_PROTOCOL_UPGRADE_VIA_SOROBAN_ENV_HOST_PREV
```

Comment:
Why is this no longer necessary?

Reply:
Because of https://github.com/stellar/stellar-core/commit/a8c5ad447d0d71002182da117f6a30a6fa0b4083